package/Dockerfile

```diff
@@ -1,4 +1,6 @@
 FROM alpine:3.12.3
-COPY bin/fleetcontroller /usr/bin/
+ARG ARCH
+ENV ARCH=$ARCH
+COPY bin/fleetcontroller-linux-${ARCH} /usr/bin/
```

Comment:
you should probably rename to `fleetcontroller` still so you don't have to change CMD in dockerfile.

Reply:
Ah this was actually a mistake and it _should_ be that. Good catch.